stable/prometheus-adapter/templates/custom-metrics-configmap.yaml

```diff
@@ -82,4 +82,32 @@ data:
 {{- if .Values.rules.custom }}
 {{ toYaml .Values.rules.custom | indent 4 }}
 {{- end -}}
+{{- if .Values.rules.resources }}
+    resourceRules:
```

Comment:
The upstream PR also changed another [few lines](https://github.com/DirectXMan12/k8s-prometheus-adapter/pull/106/files#diff-0dd77a9727079f402a22ee2312b123ca) from 5m to 1m in the default rules. We should probably change that here as well.